backbone.js

```diff
@@ -1436,8 +1436,7 @@
 
     // Are we at the app root?
     atRoot: function() {
-      var path = this.location.pathname.replace(/[^\/]$/, '$&/');
-      return path === this.root && !this.getSearch();
+      return this.location.pathname === this.root && !this.getSearch();
```

Comment:
> I felt iffy about this as well. However, I remember reading jashkenas arguing /root is not the same resource as /root/ (but I can't find it!).

I agree, though I think they should behave similarly regarding a hashChange redirect.

> I think the only downside to this as-is is: At /root and { root: '/root/' } will cause a redirect when transitioning the app to hashChange.

Something like that yea. It's a pretty nasty breaking change.